Builder/ListBuilder.php

```diff
@@ -164,7 +164,7 @@ public function buildActionFieldDescription(FieldDescriptionInterface $fieldDesc
         }
 
         if (null === $fieldDescription->getType()) {
-            $fieldDescription->setType('action');
+            $fieldDescription->setType('actions');
```

Comment:
Currently (before this PR) the bundle ignores both the `action`/`actions` types — the code chooses the template only for the `_action`-named field, and this is the essence of the bug being fixed.

Absence of any other standard action fields makes me conclude that there are no implications inside the bundle. Judging by absence of registered issues, end users don't rely on `action` or `actions` type too, so no implications there too.

This line makes the change consistent — it would be confusing to require end suers to set the type to `actions` and mention but not even use `action` internally.